extensions-core/protobuf-extensions/src/test/java/org/apache/druid/data/input/protobuf/ProtobufInputRowParserTest.java

```diff
@@ -177,6 +191,45 @@ public void testParse() throws Exception
     Assert.assertEquals(816.0F, row.getMetric("someLongColumn").floatValue(), 0.0);
   }
 
+  @Test
+  public void testParseFlattenData() throws Exception
```

Comment:
nit: same suggestion about naming, should this be `testParseFlatData` since it isn't actively flattening nested data?

Reply:
Your suggestion makes sense. I'll fix it soon.